src/inject/utils/dom.ts

```diff
@@ -173,7 +173,6 @@ export function iterateShadowHosts(root: Node, iterator: (host: Element) => void
                 return (node as Element).shadowRoot == null ? NodeFilter.FILTER_SKIP : NodeFilter.FILTER_ACCEPT;
             }
         },
-        false,
```

Comment:
Not sure what this parameter was there for. Hope nothing will break.

Reply:
I only know it's been decrapted for 2 years. 
https://github.com/Microsoft/TypeScript/blame/master/lib/lib.dom.d.ts#4766